### PR TITLE
Bump version to 1.3.3

### PR DESCRIPTION
**Pull request by gforsyth**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks Gil! 😄 Could you please merge with `master`? There are some conflicts atm

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,14 +9,15 @@ source:
   # we have to use git urls here to get the submodules needed for the
   # build
   git_url: https://github.com/dmlc/xgboost
-  git_tag: v1.3.0
+  git_tag: 000292ce6d99ed658f6f9aebabc6e9b330696e7e
```
Comment (xhochy) [inline recipe/meta.yaml:12]: Please keep the named tag
```suggestion
  git_tag: v1.3.3
```

**Comment by gforsyth** [comment]
@conda-forge/xgboost this looks ready to go!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,14 +9,14 @@ source:
   # we have to use git urls here to get the submodules needed for the
   # build
   git_url: https://github.com/dmlc/xgboost
-  git_tag: v1.3.0
+  git_tag: v1.3.3
```
Comment (jakirkham) [inline recipe/meta.yaml:12]:
```suggestion
  git_tag: v{{ version }}
```

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
Apologies merged a couple migrators that introduced conflicts. So took the liberty of fixing those conflicts here. Hope that is ok.

Giving this one last re-render then I think this is good on my end.

Thanks for putting this together Gil! 😄

**Review by jakirkham — APPROVED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**